README.md

```diff
@@ -1,20 +1,16 @@
 ### Open Learning Django Apps
 
-This repository is the home of MIT Open Learning's reusable django apps:
-
-| App | Description | Status |
-| --- | --- | --- |
-| [`mitol-django-mail`](mitol-django-mail/) | Mail app and APIs | ![CI Status](https://img.shields.io/github/workflow/status/mitodl/ol-django/ci) ![Codecov](https://img.shields.io/codecov/c/github/mitodl/ol-django?flag=mitol_django_mail) |
+This repository is the home of MIT Open Learning's reusable django apps
 
 ### Releases
 
 Changelogs are maintained according to [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 Versioning adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-Version tags follow `{package-name}-v{major}.{minor}.{patch}`
+Version tags follow `{package-name}/v{major}.{minor}.{patch}`
 To perform a release, run:
-- `invoke release APPNAME VERSION`:
+- `./pants run build-supper/bin/release -- --app APP_NAME [--bump-major] [--bump-minor] [--bump-patch] [--version VERSION]`:
```

Comment:
Typo here, looks like it should be `build-support`

Reply:
I must've been hungry...